pkgs/os-specific/darwin/apple-source-releases/xnu/default.nix

```diff
@@ -127,6 +127,8 @@ appleDerivation ({
     mkdir $out/Library/PrivateFrameworks
     mv $out/Library/Frameworks/IOKit.framework $out/Library/PrivateFrameworks
   '';
+
+  APPLE_HEADERS = builtins.readFile ./headers.txt;
```

Comment:
Why not name it appleHeaders in the original commit?

Reply:
Because it needs amending 20+ commits, I will amend them one by one later.